installer/cmd/nextgen/main.go

```diff
@@ -0,0 +1,50 @@
+package main
```

Comment:
Can we put it under just `${installer_root}/pkg`? 
I was expecting something like:

```
openshift-installer/
├── cmd
│   └── openshift-install
└── pkg
    └── graph(or asset?)
```